packages/config/src/config/base.ts

```diff
@@ -41,9 +41,9 @@ export const ConfigBase = z.object({
   /**
    * Was this configuration object generated from another object
    *
-   * @default undefined / false
+   * @default undefined
    */
-  virtual: z.boolean().optional(),
+  virtual: z.string().optional(),
```

Comment:
this would be better served as a literal type `tileset-all` `imagery-id` etc. then we can document what the virtual types are?